engines/python/setup/djl_python/huggingface.py

```diff
@@ -96,6 +96,9 @@ def get_rolling_batch_class_from_str(rolling_batch_type: str, is_mpi: bool,
     elif rolling_batch_type == "lmi-dist":
         from djl_python.rolling_batch.lmi_dist_rolling_batch import LmiDistRollingBatch
         return LmiDistRollingBatch
+    elif rolling_batch_type == "lmi-dist-v2":
```

Comment:
I think we should leave it as lmi-dist, and allow the handler to detect which version is installed.